docs/release-notes/8.18.asciidoc

```diff
@@ -0,0 +1,154 @@
+[[release-notes-header-8.18.0]]
+== 8.18
+
+[discrete]
+[[release-notes-8.18.0]]
+=== 8.18.0
+
+[discrete]
+[[known-issue-8.18.0]]
+==== Known issues
+// tag::known-issue[]
+[discrete]
+.Rules cannot be enabled if they're corrupted while upgrading from 7.17.x to 8.x 
+[%collapsible]
+====
+*Details* +
+If rule saved objects were corrupted when you upgraded from 7.17.x to 8.x, you may run into an error when turning on your rules. 
+
+*Workaround* +
+
+Duplicate your rules and enable them.
+
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.The technical preview badge incorrectly displays on the alert suppression fields for event correlation rules 
+[%collapsible]
+====
+*Details* +
+Alert suppression for event correlation rules is generally available but still shows as being in technical preview when you create a new rule.  
```

Comment:
```suggestion
On April 8, 2025, it was discovered that alert suppression for event correlation rules is incorrectly shown as being in technical preview when you create a new rule. For more information, check (https://github.com/elastic/docs-content/issues/1021)[#1021].
```

Reply:
How was this discovered in the future (tomorrow the 8th)? Perhaps you meant April 3rd?